phase3/all.jsonnet

```diff
@@ -8,4 +9,6 @@ function(cfg)
                if_enabled("heapster", (import "heapster/heapster.jsonnet")(cfg)),
                if_enabled("kube_dns", (import "kube-dns/kube-dns.jsonnet")(cfg)),
                if_enabled("weave_net", (import "weave-net/weave-net.jsonnet")(cfg)),
+               if_cni_plugin("weave", (import "weave-net/weave-net.jsonnet")(cfg)),
```

Comment:
Unfortunately resource files for cni plug resource files could are somehow linked to versions of kube* (e.g. RBAC  stuff required only if version >1.6). How do you think to manage this? Maintaining several mirror copy of weave-net.jsonnet, flannel.jsonn etc etc?

Reply:
Should we design a solution in this PR, or hold off until another one? It seems like we need a comprehensive versioning policy for this project, so I think I'd favour the latter.